.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	"recommendations": [
```

Comment:
Something is wrong with the indentation here.

Reply:
Github display bug